components/brave_ads/browser/ads_service_factory.cc

```diff
@@ -77,9 +78,19 @@ bool AdsServiceFactory::ServiceIsNULLWhileTesting() const {
 void AdsServiceFactory::RegisterProfilePrefs(
     user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref(prefs::kBraveAdsEnabled, false);
+
   registry->RegisterUint64Pref(prefs::kBraveAdsPerHour, 2);
-  registry->RegisterUint64Pref(prefs::kBraveAdsPerDay, 6);
+
+  #if defined(OS_ANDROID)
+    const int ads_per_day = 12;
```

Comment:
we have this in two places so seems like it should be in a header somewhere

Reply:
👍 fixing now

Reply:
There could be cases where the default values are different to the values set in migration paths

Reply:
migrations are likely to have different values so for future consistency did not add to a header